plugins/module_utils/turbo/lookup.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
This module won't be used by any module, this is the reason why it should be in plugin_utils, not module_utils.

See: https://github.com/ansible/ansible/pull/75254#issuecomment-880099069